java-strings/src/test/java/com/baeldung/string/WordCountUnitTest.java

```diff
@@ -0,0 +1,57 @@
+package com.baeldung.string;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+/**
+ *
+ * @author shwetaJ
+ */
+
+public class WordCountUnitTest {
+	private static final Logger logger = LoggerFactory.getLogger(WordCountUnitTest.class);
+
+	String stringIn = "I like  Cookies, Cake & Icecream but not chocolates! ";
+
+	String stringTocheck = stringIn.trim().replaceAll(" +", " ");
+
+	@Test
+	public void testCountUsingforLoop() {
+		int count = WordCount.countUsingCharAt(stringTocheck, ' ');
+		assertEquals(count, 9);
+	}
+
+	/*
+	@Test
+	public void testCountUsingSpringStringUtils() {
+		int count = WordCount.countUsingSpringStringUtils(stringTocheck, " ");
+		assertEquals(count, 9);
+	}
+	*/
```

Comment:
This is commented out. Are we going to keep it?

Reply:
Yes, it is for the reader to uncomment, I 'll add a comment for reader here.